uisdk/src/test/java/com/karhoo/uisdk/screen/rides/detail/RideDetailPresenterTest.kt

```diff
@@ -244,8 +248,8 @@ class RideDetailPresenterTest {
     @Test
     fun `observer is added to observable for guest booking`() {
         KarhooUISDKConfigurationProvider.setConfig(configuration = UnitTestUISDKConfig(context =
-                                                                                       context,
-                                                                                       authenticationMethod = AuthenticationMethod.Guest("identifier", "referer", "guestOrganisationId")))
+        context,
```

Comment:
UnitTestUISDKConfig.setGuestAuthentication(context)

Reply:
Replaced it